.setup/vagrant/setup_vagrant.sh

```diff
@@ -14,6 +14,16 @@ if [ -x "$(command -v pip3)" ]; then
     python3 ${SUBMITTY_REPOSITORY}/.setup/bin/reset_system.py
 fi
 
+OPTSTRING=":c"
```

Comment:
`-c` isn't very descriptive.  Would `--ci`, or the environment variable `CI=1` be a better option?

Reply:
This has been updated.